src/niMidi.cpp

```diff
@@ -83,6 +95,71 @@ signed char convertSignedMidiValue(unsigned char value) {
 	return value - 128;
 }
 
+//-----------------------------------------------------------------------------------------------------------------------
+// The following conversion functions (C) 2006-2008 Cockos Incorporated
+// ToDo: Eliminate those that will eventually not be used (e.g. volToChar)
+static double charToVol(unsigned char val)
+{
+	double pos = ((double)val*1000.0) / 127.0;
+	pos = SLIDER2DB(pos);
+	return DB2VAL(pos);
+
+}
+
+static unsigned char volToChar(double vol)
+{
+	double d = (DB2SLIDER(VAL2DB(vol))*127.0 / 1000.0);
+	if (d < 0.0)d = 0.0;
+	else if (d > 127.0)d = 127.0;
+
+	return (unsigned char)(d + 0.5);
+}
+
+static double charToPan(unsigned char val)
+{
+	double pos = ((double)val*1000.0 + 0.5) / 127.0;
+
+	pos = (pos - 500.0) / 500.0;
+	if (fabs(pos) < 0.08) pos = 0.0;
+
+	return pos;
+}
+
+static unsigned char panToChar(double pan)
+{
+	pan = (pan + 1.0)*63.5;
+
+	if (pan < 0.0)pan = 0.0;
+	else if (pan > 127.0)pan = 127.0;
+
+	return (unsigned char)(pan + 0.5);
+}
+// End of conversion functions (C) 2006-2008 Cockos Incorporated
+//-----------------------------------------------------------------------------------------------------------------------
+
+static unsigned char peakToChar_KkMk2(double peak)
+{
```

Comment:
nit: I'd prefer `{` on the same line as the function definition.

Reply:
agree!